inference/core/models/roboflow.py

```diff
@@ -742,26 +749,48 @@ def validate_model(self) -> None:
         if MODEL_VALIDATION_DISABLED:
             logger.debug("Model validation disabled.")
             return None
-        logger.debug("Starting model validation")
+        logger.debug(f"Starting model validation for {self.endpoint}")
+        validate_model_error_count = cache.get(
```

Comment:
do we need some mechanism for this to reset to 0 after a while?  otherwise once it hits the threshold we'll just end up early aborting with ModelArtefactError forever?

Reply:
Yea but the model itself is corrupt (and there is no way to rescue the model), so there is no point in resetting this.